src/PatientTestsApi/LoadPatient/function.json

```diff
@@ -6,7 +6,7 @@
       "direction": "in",
       "name": "req",
       "methods": [
-        "get"
+        "put"
       ],
       "route": "patient/{registration-id}"
```

Comment:
In my Pull request I chagned the registration-id field to patientId, since we don't really have any references to registrations in this version. Can we keep that consitency here? Quick find and replace all.